lib/grell/rawpage.rb

```diff
@@ -36,6 +35,17 @@ def has_selector?(selector)
       page.has_selector?(selector)
     end
 
+    def finished_all_ajax_requests?(timeout, interval)
+      Timeout::timeout(timeout) do
+        (timeout / interval).ceil.times do
+          jquery_active = page.evaluate_script('jQuery.active')
```

Comment:
`page.evaluate_script` just returns `nil` when an exception raises in the JS engine